packages/server/modules/notifications/index.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  initializeQueue,
+  consumeIncomingNotifications,
+  registerNotificationHandlers,
+  shutdownQueue
+} from '@/modules/notifications/services/queue'
+import {
+  NotificationType,
+  NotificationTypeHandlers
+} from '@/modules/notifications/helpers/types'
+import { modulesDebug } from '@/modules/shared/utils/logger'
+import { SpeckleModule } from '@/modules/shared/helpers/typeHelper'
+import { pick } from 'lodash'
+import { isTestChildServer } from '@/modules/shared/helpers/envHelper'
+
+export async function initializeConsumption(typeWhitelist?: NotificationType[]) {
+  modulesDebug('📞 Initializing notification queue consumption...')
+
+  const allHandlers: Partial<NotificationTypeHandlers> = {
+    [NotificationType.MentionedInComment]: (
+      await import('@/modules/notifications/services/handlers/mentionedInComment')
+    ).default,
+    [NotificationType.Test]: (
+      await import('@/modules/notifications/services/handlers/test')
+    ).default
+  }
+
+  registerNotificationHandlers(
+    typeWhitelist?.length ? pick(allHandlers, typeWhitelist) : allHandlers
+  )
+
+  initializeQueue()
+
+  if (isTestChildServer()) {
```

Comment:
Baking in test specific code into the production codebase is always itchy. Can we work around this somehow?

Reply:
there's no way to get around having some sort of check here, but I renamed the env var to something less test specific - DISABLE_NOTIFICATIONS_CONSUMPTION